.gitignore

```diff
@@ -135,3 +135,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# cache file
+url-status-cache
```

Comment:
If you need this great, but I'm curious what is different about your environment that it got created at the top level, instead of under `src/ingest_validation_tools` where there's already a git-ignore.

Reply:
It was actually created under `src/ingest_validation_tools` as intended but it was still being tracked. I must admit I didn't see it in the `.gitignore`.

Reply:
oh... for whatever reason, you're getting `url-status-cache` while I see `url-status-cache.db`, and the git-ignore is too strict.

For now, just keep this so this branch works for you, but I'll fix it in master, and resolve the redundancy after this is merged.